tests/python/gaia-ui-tests/gaiatest/apps/messages/regions/report.py

```diff
@@ -0,0 +1,38 @@
+import time
+
+from marionette_driver import expected, By, Wait
+from gaiatest.apps.base import Base
+from gaiatest.form_controls.header import GaiaHeader
+
+class Report(Base):
+
+    _time_sent_locator = (By.CSS_SELECTOR, 'span.sent-timestamp')
+    _time_received_locator = (By.CSS_SELECTOR, 'span.received-timestamp')
+    _header_locator = (By.ID, 'information-report-header')
+
+    _sender = (By.CSS_SELECTOR, 'a[data-name]')
+    _receiver = (By.CSS_SELECTOR, 'a.suggestion.js-contact-info')
+
+    def close_report(self):
+        GaiaHeader(self.marionette, self._header_locator).go_back()
+        from gaiatest.apps.messages.regions.message_thread import MessageThread
+        return MessageThread(self.marionette) 
+
+    def is_message_an_sms(self):
+        return self.marionette.find_element(*_text_type).get_attribute('data-l10n-id') == 'message-type-sms'
+
+    @property
+    def is_date_sent_ok(self):
+        return self.marionette.find_element(*self._time_sent_locator).get_attribute('data-l10n-date')
+        
+    @property
+    def is_date_rec_ok(self):
+        return self.marionette.find_element(*self._time_received_locator).get_attribute('data-l10n-date')
+
+    @property
+    def sender(self):
+        return self.marionette.find_element(*self._sender).text
+
+    @property
+    def tlf_receiv(self):
```

Comment:
Same comment regarding the words that are shortened. `receiver` sounds self-explanatory